build.gradle

```diff
@@ -91,6 +92,9 @@ dependencies {
   errorprone("com.google.errorprone:error_prone_core:${errorproneVersion}")
 }
 tasks.withType(JavaCompile).configureEach {
+  if (!isJava17orHigher) {
+    options.errorprone.enabled = false
+  }
```

Comment:
I think it would be more concise and clearer to write:
```
options.errorprone.enabled = isJava17orHigher
```

(The same comment goes for the Checker Framework.)

Reply:
Sounds good.